src/go/MONIT/annotationManager.go

```diff
@@ -0,0 +1,595 @@
+package main
+
+// File       : alert.go
+// Author     : Rahul Indra <indrarahul2013 AT gmail dot com>
+// Created    : Tue, 1 Sep 2020 14:08:00 GMT
+// Description: CERN MONIT infrastructure Alert CLI Tool
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+)
+
+//-------VARIABLES-------
+
+//MAX timeStamp //Saturday, May 24, 3000 3:43:26 PM
+var maxtstmp int64 = 32516091806
+
+var microSec int64 = 1000000
+
+var milliSec int64 = 1000
+
+//token name
+var token string
+
+//tags list seperated by comma
+var tags string
+
+//Action to be performed, eg. delete, update etc.
+var action string
+
+//time range
+var trange string
+
+//boolean for generating default config
+var generateConfig *bool
+
+//Config filepath
+var configFilePath string
+
+//-------VARIABLES-------
+
+//-------STRUCTS---------
+
+//Grafana Dashboard Annotation API Data struct
+type annotationData struct {
+	ID          int      `json:"id"`          //Annotation Id
+	DashboardID int      `json:"dashboardId"` //Dashboard ID
+	Created     int64    `json:"created"`     //Timestamp when the annotation created
+	Updated     int64    `json:"updated"`     //Timestamp when the annotation last updated
+	Time        int64    `json:"time"`        //Start Time of the annotation
+	TimeEnd     int64    `json:"timeEnd"`     //End Time of the annotation
+	Text        string   `json:"text"`        //Text field of the annotation
+	Tags        []string `json:"tags"`        //List of tags attached with the annotation
+}
+
+type config struct {
+	GrafanaBaseURL      string   `json:"grafanaBaseURL"`      // Grafana Annotation Base URL
+	GetAnnotationsAPI   string   `json:"getAnnotationsAPI"`   //API endpoint for fetching all annotations
```

Comment:
be consistent with commend, either use `// comment` or `//comment`, I prefer extra space behind two slashes.

Reply:
fixed